Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EntityTypes.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Entity.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace AzToolsFramework
+{
+    using EntityIdList = AZStd::vector<AZ::EntityId>;
+    using EntityIdSet = AZStd::unordered_set<AZ::EntityId>;
```

Comment:
It might be a good idea to name this EntityIdUnorderedSet since there's also AZStd::set